docs/guide/local/quick-start.md

```diff
@@ -223,6 +223,7 @@ export default defineNuxtConfig({
         endpoint: { path: '/refresh', method: 'POST' },
         refreshOnlyToken: true,
         token: {
+          refreshResponseTokenPointer: '/token',
```

Comment:
Could we move it below the `signInResponseRefreshTokenPointer` inside the docs please 😊